lib/blockchain/cardano.js

```diff
@@ -10,6 +10,12 @@ module.exports = { setup }
 
 const es = common.es
 
+function from24to12wordMnemonic(mnemonic) {
+  const entrophy = mnemonicHelpers.toEntropyBuffer(mnemonic).toString('hex')
+  const halfEntrophy = entrophy.substring(0, 32)
```

Comment:
Not sure that `halfEntrophy` is the best name.  Maybe you can just directly use it as `mnemonicHelpers.fromSeed(entrophy.subscring(0, 32))`